src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -151,6 +152,7 @@ public void start(Promise<Void> startPromise) throws Exception {
                 routerFactory.addHandlerByOperationId(this.HEALTHY.getOperationId().toString(), this.HEALTHY);
                 routerFactory.addHandlerByOperationId(this.READY.getOperationId().toString(), this.READY);
                 routerFactory.addHandlerByOperationId(this.OPENAPI.getOperationId().toString(), this.OPENAPI);
+                routerFactory.addHandlerByOperationId(this.CONFIGURATION.getOperationId().toString(), this.CONFIGURATION);
```

Comment:
I would not call it `CONFIGURATION`. For example, the version is not a configuration parameter. I would use `INFO` or something like that.

Reply:
`CONFIGURATION` etc are not `static final`, so why do they have UPPER_CASE names?

Reply:
it was in this way since the beginning, maybe for readability. We can change it but not in this PR.